styles/base/index.scss

```diff
@@ -1,2 +1,13 @@
 @use 'reset';
 @use 'elements';
+
+* {
+  box-sizing: border-box;
+}
+
+button,
+html input[type='button'],
```

Comment:
Why is the `html` ancestor needed for this particular rule?

Reply:
Yeah, I haven't had a chance to track down this history of these rules. I have the same question.